src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> STORED_FRUITS = new HashMap<>();
```

Comment:
It's not a constant, because you can add/update/remove values from it. Constant means something immutable/unchanging
```suggestion
    public static final Map<String, Integer> storedFruits = new HashMap<>();
```